checkstyle.xml

```diff
@@ -0,0 +1,211 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+          "http://checkstyle.sourceforge.net/dtds/configuration_1_3.dtd">
+
+<!--
+
+  Checkstyle configuration that checks the sun coding conventions from:
```

Comment:
Code says "sun conventions"; docs say "default Netbeans formatting settings".  Are these different, and if so which one is authoritative?

Reply:
Removed in 80334fc